tool/tctl/common/token_command.go

```diff
@@ -175,6 +176,27 @@ func (c *TokenCommand) Add(client auth.ClientI) error {
 		return trace.BadParameter("this cluster has no auth servers")
 	}
 
+	// Print signup message formatted with JSON or just print the raw token.
+	if c.format == teleport.JSON {
```

Comment:
Switch would work better here (and would be consistent with how we do it in other places).

Reply:
Sure.